charts.yaml

```
@@ -0,0 +1,4 @@
+- chart: "cilium"
+  versions: ">= 1.11.2"
+  srcHelmRepo: "https://helm.cilium.io"
+  dstCatalog: "default-test"
```

is it much harder to use URL format like `srcHelmRepo` so it is more generic?
```suggestion
  dstHelmRepo: "https://github.com/giantswarm/default-test"
```